api/package.json

```diff
@@ -73,8 +73,8 @@
     "db:delete": "rm db/dev.sqlite3 ",
     "db:reset": "npm run db:delete && npm run db:migrate && npm run db:seed",
     "test": "npm run test:api",
-    "test:api": "NODE_ENV=test npm run db:migrate && NODE_ENV=test ./node_modules/.bin/mocha --recursive --exit tests",
-    "test:api:watch": "NODE_ENV=test ./node_modules/.bin/mocha --recursive tests --watch",
+    "test:api": "NODE_ENV=test npm run db:migrate && NODE_ENV=test ./node_modules/.bin/mocha --recursive --exit --reporter dot tests",
+    "test:api:watch": "NODE_ENV=test ./node_modules/.bin/mocha --recursive tests --watch --reporter dot",
     "test:script": "NODE_ENV=test ./node_modules/.bin/mocha --recursive ./tests/unit/script",
```

Comment:
Report dot aussi sur les scripts ?

Reply:
why not. Je suis pour. Ma philosophie: moins la suite affiche de trucs et mieux je me porte. Affichage == warning ou erreur.